src/Main.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+function Main() {
+  return (
+    <div>
+        {/* fetch the data
+        images
+        current Weather
+        future Weather */}
+    </div>
```

Comment:
2 things:
- can you use a more semantic HTML element here instead of a `div`?
- And how will we pass in content as a prop, inside of `<Main>`? Remember we will be passing whole components into Main from inside of `App.js`